integrations/nifi/nifi-nar-bundles/nifi-core-bundle/nifi-core-processors/src/main/java/com/thinkbiganalytics/ingest/TableRegisterSupport.java

```diff
@@ -173,32 +166,16 @@ protected boolean createTable(String ddl) {
         return tables;
     }
 
-    public boolean registerProfileTable(String source, String tableEntity, String targetFormatOptions) {
-
-        String tableName = TableType.PROFILE.deriveQualifiedName(source, tableEntity);
-        String columnSQL = " `columnname` string,`metrictype` string,`metricvalue` string";
-        String formatSQL = TableType.PROFILE.deriveFormatSpecification("NOT_USED", targetFormatOptions);
-        String partitionSQL = TableType.PROFILE.derivePartitionSpecification(null);
-        String locationSQL = TableType.PROFILE.deriveLocationSpecification(config.pathForTableType(TableType.PROFILE), source, tableEntity);
-
-        String ddl = createDDL(tableName, columnSQL, partitionSQL, formatSQL, locationSQL, "", TableType.PROFILE.isExternal());
-        return createTable(ddl);
-    }
-
     public boolean registerStandardTables(String source, String tableEntity, ColumnSpec[] feedColumnSpecs, String feedFormatOptions, String targetFormatOptions, ColumnSpec[] partitions, ColumnSpec[]
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)